h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -232,12 +233,22 @@ public void update(double [] beta, double[] ubeta, double devianceTrain, double
     _output.setSubmodelIdx(id);
   }
 
+  protected GLMModel deepClone(Key<GLMModel> result) {
+    GLMModel newModel = IcedUtils.deepCopy(this);
+    newModel._key = result;
+    // Do not clone model metrics
+    newModel._output.clearModelMetrics(false);
+    newModel._output._training_metrics = null;
+    newModel._output._validation_metrics = null;
+    return newModel;
+  }
+
   public static class GLMParameters extends Model.Parameters {
 
     public enum MissingValuesHandling {
       MeanImputation, PlugValues, Skip
     }
-
+    static String[] CHECKPOINT_NON_MODIFIABLE_FIELDS = {};
```

Comment:
should be definitelly final, probably private as well

Reply:
Private makes it a pain to call it from GLM.java  I did add final.